thinc/optimizers.py

```diff
@@ -168,13 +173,14 @@ def __init__(
         L2_is_weight_decay (bool): Whether to interpret the L2 parameter as a
             weight decay term, in the style of the AdamW optimizer.
         """
+        self._step = 0
+        self._last_score = None
```

Comment:
Maybe a type-hint that this will be a `Tuple[int, Comparable]` in the future. Or not sure what to put in place of `Comparable` , maybe its always just a `float` dunno. https://github.com/python/typing/issues/59

Reply:
The type hint is already above in the definition of member variables.